### DISCUSSION TITLE
Force Layout with Network Graph? (@visx/network)

### DISCUSSION THREAD

**Discussion by GregLahaye** [airbnb/visx#1134] [Q&A]
Hi all, I have data in the form of nodes connected to each other and I am trying to display them as a network graph with @visx/network but I am unsure how to go about calculating positions of the nodes. I'm trying to get a result that looks something like d3-force (though I don't need animations). Thanks for any help.

**Comment by hyfydistro** [comment]
I was wondering about this too. I thought about using some circumference function but I don't really know either what's the tried and best methods.
